bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/handler/GardenaAccountHandler.java

```diff
@@ -51,80 +64,188 @@
+        apiCallSuppressionEnd = "".equals(propertyValue) ? Instant.MIN : Instant.parse(propertyValue);
+    }
+
+    /**
+     * Get the duration remaining until the end of the api call suppression window, or Duration.ZERO if we are outside
+     * the call suppression window.
+     *
+     * @return the duration until the end of the suppression window, or zero.
+     */
+    private Duration getApiCallSuppressionDurationRemaining() {
+        Instant now = Instant.now();
+        if (now.isAfter(apiCallSuppressionStart) && now.isBefore(apiCallSuppressionEnd)) {
+            return Duration.between(now, apiCallSuppressionEnd);
+        }
+        return Duration.ZERO;
+    }
+
+    /**
+     * Updates the time when api call suppression begins to now(), and the time when api call suppression ends to now()
+     * plus the given delay. The delay must not be a negative duration. Saves the start and end time values as
+     * properties to ensure consistent behaviour across restarts.
+     *
+     * @param delay the duration between the start and end of the suppression window.
+     */
+    private void apiCallSuppressionPropertiesUpdate(Duration delay) {
+        Duration offset = delay.isNegative() ? Duration.ZERO : delay;
+        apiCallSuppressionStart = Instant.now();
+        apiCallSuppressionEnd = apiCallSuppressionStart.plus(offset);
+        Thing thing = getThing();
+        thing.setProperty(GardenaBindingConstants.API_CALL_SUPPRESSION_START, apiCallSuppressionStart.toString());
+        thing.setProperty(GardenaBindingConstants.API_CALL_SUPPRESSION_END, apiCallSuppressionEnd.toString());
     }
 
     @Override
     public void initialize() {
         logger.debug("Initializing Gardena account '{}'", getThing().getUID().getId());
-        initializeGardena();
+        loadApiCallSuppressionProperties();
+        Duration delay = getApiCallSuppressionDurationRemaining();
+        if (Duration.ZERO.equals(delay)) {
+            // do immediate initialisation
+            scheduler.submit(() -> initializeGardena());
+        } else {
+            // delay the initialisation
+            scheduleReinitialize(delay);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, getUiText());
+        }
     }
 
     public void setDiscoveryService(GardenaDeviceDiscoveryService discoveryService) {
         this.discoveryService = discoveryService;
     }
 
+    /**
+     * Format a localised description text to display on the thing main UI page.
+     *
+     * @return the (localised) description text.
+     */
+    private String getUiText() {
+        String template = i18nProvider.getText(bundle, RECONNECT_MSG_KEY, RECONNECT_MSG_KEY,
+                localeProvider.getLocale());
+        String dateTime = LocalDateTime.ofInstant(apiCallSuppressionEnd, timeZoneProvider.getTimeZone())
+                .truncatedTo(ChronoUnit.SECONDS).format(DateTimeFormatter.ISO_LOCAL_DATE_TIME);
```

Comment:
Any specific reason for not using a localized date/time format here? Additionally, one could omit the date if the suppression time span is less than 24 hours.

Reply:
> reason for not using a localized date/time format

No reason. Which do you suggest?

> could omit the date if the suppression time span is less than 24 hours

Agreed. Will do.

Reply:
> No reason. Which do you suggest?

See my [earlier comment](https://github.com/openhab/openhab-addons/pull/13016#discussion_r928720621) ;-)

Reply:
Ok. Done.